src/pseudopeople/interface.py

```diff
@@ -111,7 +111,7 @@ def _generate_dataset(
         try:
             from distributed.client import default_client
 
-            default_client().run(lambda: configure_logging_to_terminal(verbose))
+            default_client().run(lambda: configure_logging_to_terminal(verbose))  # type: ignore [no-untyped-call]
```

Comment:
this is an external package.

Reply:
Are you sure it's because it's an external package and not because it's a lambda function?

Reply:
The error was:
`src/pseudopeople/interface.py:114: error: Call to untyped function "default_client" in typed context  [no-untyped-call]` so i think so.

Reply:
If this truly is the case (and not the lambda) the propery way is to first check if the package has an external typing stubs to install ([example](https://github.com/ihmeuw/layered_config_tree/blob/main/setup.py#L63-L65)). 

If it doesn't then instead of line-by-line ignoring you can wholesale ignore any untyped packages via the pyproject.toml [[tool.mypy.overrides]] `module` list at the bottom.

Reply:
I couldn't find any stubs for `distributed`. I'll add to the overrides instead, ty

Reply:
Hm, that didn't seem to work.